ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/InitializeMigrationCommand.java

```diff
@@ -71,30 +73,41 @@ private String createVersionTable(final String name, final String topic) {
 
   @Override
   protected int command() {
-    final MigrationConfig properties;
+    if (!validateConfigFilePresent()) {
+      return 1;
```

Comment:
I don't think it's important that the config file name is configurable. I think it's fine for us to assume a strict directory structure, otherwise we'll also have to allow users to configure the migrations directory name, and so on. We can always add a feature to make this directory structure more configurable in the future if we want; it seems preferable to keep stricter assumptions now and prioritize releasing a minimal version of the tool to get users up and running.

Reply:
In terms of the user experience of always having to pass in the `--config-file` option for all commands, it might be good to read from an environment variable and/or use `./ksql-migrations.properties` (i.e., assume that the tool is being invoked from the directory itself) if it's not specified. We can do this as a follow-up.

Reply:
Ohh okay so the purpose of this is to identify the directory that the config file and /migrations is in. IMO it might make more sense to just pass the directory, but that's a pretty minor point that can be done as a follow-up.

Reply:
Yeah exactly, but I actually think it makes sense to pass in the config file rather than the directory. Besides feeling more natural, it also allows us to support configurable directory structures in the future.